handsontable/src/shortcuts/__tests__/utils.unit.js

```diff
@@ -89,6 +89,168 @@ describe('utils', () => {
 
       expect(key).toBe('slash');
     });
+
+    it('should get canonical form from the numpad 0', () => {
+      const key = normalizeEventKey({ key: '0', code: 'Numpad0', which: 96 });
+
+      expect(key).toBe('numpad0');
+    });
+
+    it('should get canonical form from the numpad 1', () => {
+      const key = normalizeEventKey({ key: '1', code: 'Numpad1', which: 97 });
+
+      expect(key).toBe('numpad1');
+    });
+
+    it('should get canonical form from the numpad 2', () => {
+      const key = normalizeEventKey({ key: '2', code: 'Numpad2', which: 98 });
+
+      expect(key).toBe('numpad2');
+    });
+
+    it('should get canonical form from the numpad 3', () => {
+      const key = normalizeEventKey({ key: '3', code: 'Numpad3', which: 99 });
+
+      expect(key).toBe('numpad3');
+    });
+
+    it('should get canonical form from the numpad 4', () => {
+      const key = normalizeEventKey({ key: '4', code: 'Numpad4', which: 100 });
+
+      expect(key).toBe('numpad4');
+    });
+
+    it('should get canonical form from the numpad 5', () => {
+      const key = normalizeEventKey({ key: '5', code: 'Numpad5', which: 101 });
+
+      expect(key).toBe('numpad5');
+    });
+
+    it('should get canonical form from the numpad 6', () => {
+      const key = normalizeEventKey({ key: '6', code: 'Numpad6', which: 102 });
+
+      expect(key).toBe('numpad6');
+    });
+
+    it('should get canonical form from the numpad 7', () => {
+      const key = normalizeEventKey({ key: '7', code: 'Numpad7', which: 103 });
+
+      expect(key).toBe('numpad7');
+    });
+
+    it('should get canonical form from the numpad 8', () => {
+      const key = normalizeEventKey({ key: '8', code: 'Numpad8', which: 104 });
+
+      expect(key).toBe('numpad8');
+    });
+
+    it('should get canonical form from the numpad 1', () => {
```

Comment:
`numpad 1` test is duplicated. `numpad 9` test is missing

Reply:
Fixed within 53db29d3c9f1566abb5018036184b0a4f84aee5d.